shell/platform/darwin/ios/framework/Source/FlutterView.mm

```diff
@@ -21,9 +22,10 @@
 #include "flutter/shell/platform/darwin/ios/ios_surface_metal.h"
 #endif  //  FLUTTER_SHELL_ENABLE_METAL
 
-@implementation FlutterView
-
-id<FlutterViewEngineDelegate> _delegate;
+@implementation FlutterView {
```

Comment:
Can we file a bug to audit the codebase for existing issues like this? I the `_delegate` being static got past code review, there may be more instance of this kind of misuse.

Reply:
https://github.com/flutter/flutter/issues/39933